dev/integration_tests/release_smoke_test/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -258,24 +257,6 @@
 			shellScript = "diff \"${PODS_PODFILE_DIR_PATH}/Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n# This output is used by Xcode 'outputs' to avoid re-running this script phase.\necho \"SUCCESS\" > \"${SCRIPT_OUTPUT_FILE_0}\"\n";
 			showEnvVarsInLog = 0;
 		};
-		E2C7C0A25D741282716C3051 /* [CP] Embed Pods Frameworks */ = {
```

Comment:
CocoaPods removed this since it's no longer embedding and code signing the Flutter.framework.